setup.py

```diff
@@ -1,31 +0,0 @@
-from setuptools import find_packages, setup
-
-import versioneer
-
-with open("requirements.txt", "rt") as fp:
-    install_requires = [
-        line for line in fp.read().splitlines()
-        if line and not line.startswith("#")
-    ]
-
-
-setup(
-    name="lightpath",
-    version=versioneer.get_version(),
-    cmdclass=versioneer.get_cmdclass(),
-    license="BSD",
-    author="SLAC National Accelerator Laboratory",
-    packages=find_packages(),
-    package_data={
-        "lightpath": ["ui/*.ui"],
```

Comment:
Double-checking that `ui/*.ui` gets included in the source distribution and wheel:
```
opying lightpath/ui/__init__.py -> lightpath-1.0.2.dev36+g22b74e9/lightpath/ui
copying lightpath/ui/device.ui -> lightpath-1.0.2.dev36+g22b74e9/lightpath/ui
copying lightpath/ui/gui.py -> lightpath-1.0.2.dev36+g22b74e9/lightpath/ui
copying lightpath/ui/lightapp.ui -> lightpath-1.0.2.dev36+g22b74e9/lightpath/ui
```

👍